esmvalcore/config/extra_facets/native6-mappings.yml

```diff
@@ -0,0 +1,192 @@
+# Extra facets for native6 data
```

Comment:
It would be nice to have `era5` in the filename, to make it clear that this is about ERA5 data. How about `native6-era5.yml`?

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2178/commits/4c3f1beb0e55fa70d0e7c9bdfcc1096231ecce12